keanu-python/keanu/keanu_random.py

```diff
@@ -22,3 +23,8 @@ def next_double(self) -> float:
     @staticmethod
     def set_default_random_seed(seed: int) -> None:
         k.jvm_view().KeanuRandom.setDefaultRandomSeed(seed)
+
+
+def set_deterministic_state():
+    KeanuRandom.set_default_random_seed(0)
+    k.jvm_view().VertexId.resetIdGenerator()
```

Comment:
can we write a quick test to make sure it resets the Id as expected? Or have we got one on the Java side that covers it

Reply:
Good suggestion - I'll knock up a super quick test